modules/mobile/spec/request/immunizations_request_spec.rb

```diff
@@ -471,6 +471,78 @@
           )
         end
       end
+
+      context 'when cvx_code is missing' do
+        let(:immunizations_request_missing_cvx) do
+          VCR.use_cassette('lighthouse_health/get_immunizations_cvx_code_missing', match_requests_on: %i[method uri]) do
+            get '/mobile/v0/health/immunizations', headers: iam_headers, params: nil
+          end
+        end
+
+        it 'increments statsd' do
+          expect do
+            immunizations_request_missing_cvx
+          end.to trigger_statsd_increment('mobile.immunizations.cvx_code_missing', times: 1)
+        end
+
+        it 'returns a 200' do
+          immunizations_request_missing_cvx
+          expect(response).to have_http_status(:ok)
+        end
+
+        it 'sets cvxCode to nil' do
+          immunizations_request_missing_cvx
+          expect(response.parsed_body['data'][0]['attributes']).to eq(
```

Comment:
This test and the matching one below are susceptible to churn if the logic behind any other attributes change. For example, if we add or remove an attribute, these tests will need to be adjusted. It might be better to just test the specific attribute we care about like:
`expect(response.dig('data', 0, 'attributes', 'cvxCode').to be_nil`

Reply:
Good point. I made this change but for cvxcode I kept the manufacturer and group name check for nil since those are derived from the cvxcode.